File: reverse-dictionary/reverse-dictionary.ts
let asciiDict: { [key: string]: number } = {'A': 65, 'B': 66, 'C': 67, 'D': 68}

let reversedDict: { [key: number]: string } = Object.fromEntries(
    Object.entries(asciiDict).map(([key, value]) => [value, key])
)

console.log(reversedDict)

File: reverse-dictionary/reverse-dictionary.go
package main

import "fmt"

func main()  {
	asciiDict := map[string]int{"A": 65, "B": 66, "C": 67, "D": 68}
	reversedDict := make(map[int]string)

	for key, value := range asciiDict {
		reversedDict[value] = key
	}

	fmt.Println(reversedDict)
}